github-status/handler.go

```diff
@@ -128,6 +128,21 @@ func buildPublicStatusURL(status string, statusContext string, event *sdk.Event)
 				url = publicURL
 			}
 		}
+	} else if status == sdk.StatusFailure {
+		publicURL := os.Getenv("gateway_public_url")
+		gatewayPrettyURL := os.Getenv("gateway_pretty_url")
```

Comment:
Quick question.. have you tested the code for `gateway_pretty_url`? I just need to know that works before merging/releasing.

Reply:
Deploy the router component and then set an alias in your hosts file to make use of it. Then provide `gateway_pretty_url` in the YAML

Reply:
I have not tested that particular feature, I will try that and test it.

Reply:
tested for `pretty_url` as well, against same repo and it is working fine.